cvat/apps/engine/management/commands/migrateredis.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2025 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import importlib.util as importlib_util
+import sys
+from pathlib import Path
+from typing import cast
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from cvat.apps.engine.models import RedisMigration
+from cvat.apps.engine.redis_migrations import BaseMigration
+
+
+def get_migration_class(module_name: str, file_path: Path) -> BaseMigration:
+    spec = importlib_util.spec_from_file_location(module_name, file_path)
+    module = importlib_util.module_from_spec(spec)
+    spec.loader.exec_module(module)
+    MigrationClass = getattr(module, "Migration", None)
+    if not MigrationClass or not issubclass(MigrationClass, BaseMigration):
+        raise Exception(f"Invalid migration: {file_path}")
+
+    return MigrationClass
+
+
+class Command(BaseCommand):
+    help = "Applies Redis migrations and records them in the database"
+
+    def handle(self, *args, **options) -> None:
+        migrations_dir = Path(settings.REDIS_MIGRATIONS_ROOT)
```

Comment:
This seems a bit inflexible, because it only allows for one migration root. What will we do if we need to add migrations to the Enterprise repository, for example?

The database migration functionality in Django loads migrations from each app; perhaps we could do the same here?

Reply:
Good point, fixed